base/body.go

```diff
@@ -45,6 +45,30 @@ func ReadBody(ds *dataset.Dataset, format dataset.DataFormat, fcfg dataset.Forma
 	return data, nil
 }
 
+func ReadBodyAsInterface(ds *dataset.Dataset, limit, offset int, all bool) (interface{}, error) {
```

Comment:
Considering we should rename this to `GetBody` & rename the current `ReadBody` to `ReadBodyBytes`, or if we think it is self explanatory, keep it as `ReadBody`